charts/gardener/provider-local/templates/deployment.yaml

```
@@ -64,11 +68,17 @@ spec:
         - --backupbucket-container-mount-path={{ .Values.controllers.backupbucket.containerMountPath }}
         - --heartbeat-namespace={{ .Release.Namespace }}
         - --heartbeat-renew-interval-seconds={{ .Values.controllers.heartbeat.renewIntervalSeconds }}
+        {{- if .Values.gardener.runtimeCluster.enabled }}
+        - --disable-webhooks="*"
+        - --controllers=backupbucket_controller,dnsrecord
```

```suggestion
        - --controllers=backupbucket,dnsrecord
```

Shouldn't it be like this?  Please rename the controller name in `pkg/provider-local/controller/backupbucket/add.go`